src/test/java/io/managed/services/test/rhoas/KafkaRhoasAclTests.java

```diff
@@ -0,0 +1,245 @@
+        LOGGER.debug(kafka);
+    }
+
+    @AfterClass(alwaysRun = true)
+    @SneakyThrows
+    public void clean() {
+        var kafkaMgmtApi =  KafkaMgmtApiUtils.kafkaMgmtApi(Environment.OPENSHIFT_API_URI, Environment.PRIMARY_OFFLINE_TOKEN);
+        var securityMgmtApi = SecurityMgmtAPIUtils.securityMgmtApi(Environment.OPENSHIFT_API_URI, Environment.PRIMARY_OFFLINE_TOKEN);
+        
+        if (Environment.SKIP_KAFKA_TEARDOWN) {
+            LOGGER.warn("skip kafka instance clean up");
+            return;
+        }
+
+        try {
+            KafkaMgmtApiUtils.deleteKafkaByNameIfExists(kafkaMgmtApi, KAFKA_INSTANCE_NAME);
+        } catch (Throwable t) {
+            LOGGER.error("delete kafka instance error: ", t);
+        }
+
+        try {
+            SecurityMgmtAPIUtils.cleanServiceAccount(securityMgmtApi, SERVICE_ACCOUNT_NAME);
+        } catch (Throwable t) {
+            LOGGER.error("delete service account error: ", t);
+        }
+
+        try {
+            LOGGER.info("logout user from rhoas");
+            cli.logout();
+        } catch (Throwable t) {
+            LOGGER.error("CLI logout error: ", t);
+        }
+
+        try {
+            LOGGER.info("delete workdir: {}", cli.getWorkdir());
+            FileUtils.deleteDirectory(new File(cli.getWorkdir()));
+        } catch (Throwable t) {
+            LOGGER.error("clean workdir error: ", t);
+        }
+
+        bwait(vertx.close());
+    }
+    
+    @DataProvider(name = "aclEntityPairs")
+    public Object[][] aclEntityPairsDataProvider() {
+        return new Object[][]{
+            {ACLEntityType.USER, EXTERNAL_USER},
+            {ACLEntityType.SERVICE_ACCOUNT, SERVICE_ACCOUNT_NAME}
+        };
+    }
+
+    // Primary user creates specific ACL (details not important) which allows entity to perform all operation upon specific topic
+    // Outcome: ACL assigned to entity is created and can be found once it is searched for based on entity
+    @SneakyThrows
+    @Test(dataProvider = "aclEntityPairs", priority = 1)
+    public void testCreateAllAllowAcl(ACLEntityType entityType, String entityIdentificator) {
+        LOGGER.info("Create ACL for {}", entityType.name);
+        cli.createAcl(entityType, entityIdentificator, AclOperation.ALL, AclPermissionType.ALLOW, TOPIC_NAME);
+        var aclList = cli.listACLs(entityType, entityIdentificator);
+        Optional<AclBinding> aclFiltered = CLIUtils.searchAcl(aclList, entityIdentificator, AclOperation.ALL, AclPermissionType.ALLOW, AclResourceType.TOPIC, TOPIC_NAME);
+        assertTrue(aclFiltered.isPresent(), "ACL created must exist");
+    }
+
+    // Primary user deleted specific ACL (details not important) upon specific topic
+    // Outcome: ACL assigned to entity is remove and can NOT be found once it is searched for based on entity
+    @SneakyThrows
+    @Test(dataProvider = "aclEntityPairs", dependsOnMethods = "testCreateAllAllowAcl", enabled = true)
```

Comment:
we can get rid of priority and dependency here as well. although dependsOnMethdods is good approach, we (internally) also depends on fact tht this test method will go right after that method (ue to cleanups we use) So i suggest to reimplement this method to get rid of this dependency completely. 
```suggestion
    @Test(dataProvider = "aclEntityPairs")
```

Reply:
As I said in previous conversation: I prefer to have 2 different tests: create and delete ACLs.